sdk/ml/azure-ai-ml/tests/automl_job/e2etests/test_remote_text_classification.py

```diff
@@ -47,6 +40,8 @@ def test_remote_run_text_classification(
 
         job.set_limits(timeout_minutes=60, max_concurrent_trials=1)
         job.set_featurization(dataset_language="eng")
+        if components:
+            job.set_training_parameters(model_name="microsoft/deberta-large-mnli")
```

Comment:
why are you setting this twice?

Reply:
oops, looks like NLP tests were already good to go!